apps/web/src/app/test-kyc/test-script.ts

```
@@ -0,0 +1,47 @@
+  const statuses: ('pending' | 'approved' | 'rejected')[] = ['pending', 'approved', 'rejected'];
+  
+  for (const status of statuses) {
+    console.log(`Setting status to: ${status}`);
+    try {
+      const response = await fetch('/api/test/kyc-update', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          user_id: 'test-user-' + Math.random().toString(36).slice(2, 7),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)